### DISCUSSION TITLE
AllowFreeInput in DatePickerInput v6/v7

### DISCUSSION THREAD

**Discussion by gius** [mantinedev/mantine#4717] [Q&A] [answered]
Hi, is it possible to directly input dates using keyboard like with https://v5.mantine.dev/dates/date-picker/#allow-free-input in v5?

**Comment by rtivital** [comment, marked as answer]
- https://mantine.dev/dates/date-input/
- https://v7.mantine.dev/dates/date-input

**Comment by gius** [comment]
🤦 ah, so it is like this:

DateInput = <DatePicker allowFreeInput
DatePickerInput = <DatePicker allowFreeInput={false}

I must have missed that :-( 
Thank you for the quick response!
